src/test-kit/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const INPUT = "#commentInput";
+export const POSTCOMMENT = "#postComment";
```

Comment:
Post-comment-what?

Maybe be more explicit, like POST_BUTTON or POST_COMMENT_BUTTON